### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by hjacobs**
* update docs from internal repo
* add links and images for "Running Kubernetes in Production"

### CODE REVIEW

**Comment by hjacobs** [comment]
The contents of this PR can be seen rendered on https://kubernetes-on-aws.readthedocs.io/en/update-docs/admin-guide/kubernetes-in-production.html

**Comment by hjacobs** [comment]
:+1:

File: docs/user-guide/running-in-production.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        env:
+          # set the maximum available memory as JVM would assume host/node capacity otherwise
+          # this is evaluated by java-dynamic-memory-opts in the Zalando OpenJDK base image
+          # see https://github.com/zalando/docker-openjdk
+          - name: MEM_TOTAL_KB
+            valueFrom:
+              resourceFieldRef:
+                resource: limits.memory
+                divisor: 1Ki
+		resources:
```
Comment (mikkeloscar) [inline docs/user-guide/running-in-production.rst]: TAB :)

Comment (szuecs) [inline docs/user-guide/running-in-production.rst]: @mikkeloscar @hjacobs fixed

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
@mikkeloscar you tab issue was fixed

**Comment by mikkeloscar** [comment]
:+1:

File: docs/user-guide/running-in-production.rst (hunk 1 above)
Comment (hjacobs) [inline docs/user-guide/running-in-production.rst]: Arrr, my VIM does not use TABs, I wonder how this could happen! **angry at myself** :smirk:

**Pull request merged**